BlueprintUI/Sources/Layout/EqualSizeStack.swift

```diff
@@ -0,0 +1,134 @@
+/// An element that sizes all of its children equally, stacking them according to
+/// the specified `direction` and spacing them according to the specified `gutter`.
```

Comment:
It looks like the cross-axis sizing behavior is to justify all children to the full width/height of the stack. Can you add that to the docs?

Reply:
I would also mention the measuring behavior: even though it sizes all children equally, its measuring implementation returns the sum of all child sizes along the primary axis.

I'm actually not sure that the current measuring behavior is particularly helpful, as the layout implementation will not take the child sizes into account when assigning layout attributes. I think we may want this implementation to simply take up all available space? I'd have to play with this to form a stronger opinion, which I have not done.

Reply:
I could also see an argument that it could take the size of the largest child and multiply that by the number of children, since that would be the smallest size that would layout to fit all children. But measuring the be the maximum works too.